style.css

```diff
@@ -1,11 +1,11 @@
 .timeline-pf-zoom-area {
-  fill: transparent;
   cursor: pointer;
+  fill: transparent;
 }
 
 .timeline-pf-y-tick {
-  stroke: black;
   fill: none;
+  stroke: black;
```

Comment:
Would be nice to use less and include patternfly variables for colors. Not in the scope of this PR though.